doc/setting_up_sidewalk_environment/setting_up_sdk.rst

```diff
@@ -12,51 +12,55 @@ Once confirmed, follow the `Installing the nRF Connect SDK`_ instructions.
 
 .. _dk_building_sample_app:
 
-Downloading the Sidewalk repository
+Setting up the Sidewalk Repository
 ***********************************
 
-Once you have installed the nRF Connect SDK, download the Sidewalk application:
+The Sidewalk repository is now managed through its own west.yml configuration. To set up the Sidewalk development environment:
 
-1. Open a terminal window in the ncs folder.
-   Your directory structure should look as follows:
+1. Clone the Sidewalk repository:
 
    .. code-block:: console
 
-       .
-       |___ .west
-       |___ bootloader
-       |___ modules
-       |___ nrf
-       |___ nrfxlib
-       |___ zephyr
-       |___ ...
+      git clone https://github.com/nordicsemiconductor/sidewalk.git sidewalk
```

Comment:
I think you can initlize west form sidewalk repostoriy directly, no need to clone it with git and init west locally.

Reply:
hmm true, you can do i both ways, I will try to add it to the doc